### PR TITLE
Mongolid "support"

### PR DESCRIPTION
**Pull request by ravanscafi**

### CODE REVIEW

File: src/AuthCode.php [hunk 1]
```diff
@@ -33,10 +32,10 @@ class AuthCode extends Model
     /**
      * Get the client that owns the authentication code.
      *
-     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     * @return Client|null
```
Comment (gmsantos) [inline src/AuthCode.php:17]: Laravel uses FQN on docblocks

Comment (ravanscafi) [reply src/AuthCode.php:17]: e eu lá to ligando pra isso? hahaha

**Review by gmsantos — APPROVED** [review]

File: src/Http/Controllers/AuthorizedAccessTokenController.php [hunk 2]
```diff
@@ -37,8 +37,8 @@ public function forUser(Request $request)
     {
         $tokens = $this->tokenRepository->forUser($request->user()->getKey());
 
-        return $tokens->load('client')->filter(function ($token) {
-            return ! $token->client->firstParty() && ! $token->revoked;
+        return collect($tokens)->filter(function ($token) {
+            return ! $token->revoked && ! $token->client()->firstParty();
```
Comment (djonasm) [inline src/Http/Controllers/AuthorizedAccessTokenController.php:41]: return !$token->revoked && !$token->client()->firstParty();

Comment (ravanscafi) [reply src/Http/Controllers/AuthorizedAccessTokenController.php:41]: ta seguindo o PSR do laravel

**Review by djonasm — APPROVED** [review]

File: src/Http/Controllers/AuthorizedAccessTokenController.php (hunk 2 above)
Comment (gmsantos) [reply src/Http/Controllers/AuthorizedAccessTokenController.php:41]: *codestyle

[Taylor não curte psr](https://github.com/laravel/framework/pull/20737#issuecomment-324907441).

**Pull request merged**